daemons/compute/server/version/version.go

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2023 Hewlett Packard Enterprise Development LP
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package version
+
+// The current data-mover version string. Clients can use this to identify particular
+// information about the installed data-mover daemon.
+var (
+	// Version contains the current version of the data mover daemon.
+	version = "v0.0.1"
```

Comment:
Can this eventually get pulled from kustomization.yaml (or somewhere)? Perhaps the same thing with the commitHash below?

Reply:
This is the version of the data-mover daemon; kubernetes is not involved. Are you asking for a kuberntes data-movement resource value?

`commitHash` comes from the git commit. I'm not sure what you're asking here.